api/external/secrets/secrets.go

```diff
@@ -166,8 +167,13 @@ func mapToKvs(m map[string]string) []*Kv {
 // will not add an error and be considered valid.
 func requiredField(value string, message string, errorList []*error) []*error {
 	if value == "" {
+<<<<<<< HEAD
 		newErrors := append(make([]*error, 0), errorList...)
 		err := errors.New(message)
+=======
+		newErrors := append(make([]*error, 0), errors...)
+		err := errorutils.ProcessInvalid(nil, message)
+>>>>>>> Refactor duplicated utils to common lib
```

Comment:
I think you've conflicted with my recent cleanup to just rip out this dependency from this location

Reply:
Whoops, I missed this one. I saw your cleanup, and am wondering about InvalidSecretError. I wanted to take it out of the FormatErrorMsg func so that func can be shared but am not sure where to put it or if we can remove it.

Reply:
I didn't go that route just because I looked at a lot of that error handling code and wasn't convinced it is really the kind of code we want to share via lib.  It is really only duplicated because we happen to copy/pasta services out of compliance.  That said, as I'm not the one typically working in these services, I'm happy with whatever y'all choose.